addons/point_of_sale/static/src/app/generic_components/numpad/numpad.js

```diff
@@ -33,10 +33,10 @@ export function getButtons(env, rightColumn) {
         { value: "8" },
         { value: "9" },
         ...(rightColumn ? [rightColumn[2]] : []),
-        ...(rightColumn ? [rightColumn[3]] : []),
-        { value: "0" },
         { value: env.services.localization.decimalPoint },
+        { value: "0" },
         { value: "Backspace", text: "⌫" },
+        ...(rightColumn ? [rightColumn[3]] : []),
     ];
```

Comment:
no need for this change

Reply:
Thanks

Reply:
this change still has to be reverted. The numpad changed in master after the printscreen was taken for the spec. That's why they differ, not intentionally